### ISSUE TITLE
Create ftp connection to Mega.io 

### ISSUE THREAD

**Issue by JuliusFabini** [meganz/MEGAcmd#713]
Hello, 

I am paying user of mega.io for company. I would like to use mega for a server backup with ftp connection on server control panel I see this option: 

![grafik](https://user-images.githubusercontent.com/98028285/189477218-aa8ee7b9-5d22-433f-8d0b-88458580a0bd.png)

Can I connect it with an folder on mega.io ? 

THank you kind regards 
Julius

**Comment by polmr** [comment]
Yes provided you install MEGAcmd and have it serving a folder via FTP. See: https://github.com/meganz/MEGAcmd/blob/master/contrib/docs/FTP.md
